### DISCUSSION TITLE
Error 408 Request Timeout while uploading large files

### DISCUSSION THREAD

**Discussion by AlphaTrash** [directus/directus#17992] [Archive]
### Describe the Bug

Directus version : 9.23.4

While trying to upload large files (30GB for example) on Directus with the interface, this error appears : 
`
{
  "stack": "AxiosError: Network Error\n    at R.onerror (http://localhost:8055/admin/assets/use-sync-142c55ed.js:29:4380)",
  "message": "Network Error",
  "name": "AxiosError",
  "code": "ERR_NETWORK",
  "config": {}
}`

When trying to upload files through the API, the following error appears:
Error 408 (Request Timeout)  

In both cases, the issue appears only for files that take more than ~5.30min to be uploaded.

### To Reproduce

You can reproduce the issue by trying to upload a file that take more than 6 minutes to be fully uploaded either using the interface of your Directus project or using the API.

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by rijkvanzanten** [comment]
This to me feels like a technical limitation of exact configuration and hosting environment. Most proxies/cdns will timeout way before that mark, so I'm already kinda surprised it made it that long!

**Reply by AlphaTrash** [reply]
I don't use any cdn or proxy, I'm only working in local on my computer...

**Comment by domdus** [comment]
Are you using NGINX? I faced such an issue in the past, maybe this helps:
https://github.com/directus/directus/issues/15769#issuecomment-1263507183

**Reply by AlphaTrash** [reply]
I don't use any proxy

**Comment by RegisWeduLabs** [comment]
+1 - having the same issue when uploading a file that takes 5m+ to upload.  I've set:

SERVER_KEEP_ALIVE_TIMEOUT=3600000
SERVER_HEADERS_TIMEOUT=3600000 

And it works fine now.  Might be a little long depending on the use case.

**Comment by delivey** [comment]
Still getting this issue on local environment.
